keel-clouddriver/src/main/kotlin/com/netflix/spinnaker/keel/clouddriver/model/NamedImage.kt

```diff
@@ -34,7 +34,8 @@ val NamedImage.creationDate: Instant
     }
 
 val NamedImage.hasAppVersion: Boolean
-  get() = tagsByImageId
+  get() = tagsByImageId.values.isNotEmpty() &&
```

Comment:
I think you could combine the two statements as:
```kotlin
tagsByImageId
  .values
  .let { vals ->
    vals.isNotEmpty() && vals.all { it != null && it.containsKey(“appversion”) }
  }
```

Reply:
Good call. Updated in 5f16c035